### ISSUE TITLE
Question about callback 

### ISSUE THREAD

**Issue by hellzakk** [perak/meteor-mqtt-collection#6]
Hi @perak this is just a question, i have understand how the "normal" callback functions work but i dont understand how the mqtt-collection callback function work and when its called...
Can you give a hint?

**Comment by perak** [comment]
@hellzakk sorry, I don't understand. Which callback... can you rephrase your question?

**Comment by hellzakk** [comment]
@perak i completely missed the right project :smile: 
sorry, can you delete this useless waste of time? :D

**Comment by perak** [comment]
:D
